main.go

```diff
@@ -325,3 +325,15 @@ func initSelfNodeRemediationAgent(mgr manager.Manager) {
 		os.Exit(1)
 	}
 }
+
+func calcNumOfBatches(myPeers *peers.Peers) time.Duration {
+	// 2. time for asking peers (10% batches + 1st smaller batch)
+	numberOfBatches := 10
+	numOfWorkerPeers := len(myPeers.GetPeersAddresses(peers.Worker))
```

Comment:
this will always return 0, because Start() wasn't called yet

Reply:
Thanks, Great catch